src/app/space/app-launcher/services/app-launcher-dependency-check.service.ts

```diff
@@ -23,7 +23,8 @@ export class AppLauncherDependencyCheckService implements DependencyCheckService
         groupId: 'io.openshift.booster',
         projectName: '',
         projectVersion: '1.0.0',
-        spacePath: '/' + (this.context ? (this.context.space ? this.context.space.id : '') : '')
+        spacePath: '/' + (this.context ? (this.context.space ? this.context.space.id : '') : ''),
```

Comment:
didn't we need to change `spacePath` to `space`?

Reply:
No, in the dependency check model it is still ```spacePath```.